lib/workers/global/config/parse/env.ts

```diff
@@ -41,6 +41,8 @@ const renameKeys = {
   gitLabAutomerge: 'platformAutomerge', // migrate: gitLabAutomerge
   mergeConfidenceApiBaseUrl: 'mergeConfidenceEndpoint',
   mergeConfidenceSupportedDatasources: 'mergeConfidenceDatasources',
+  allowPostUpgradeCommandTemplating: 'allowCommandTemplating',
+  allowedPostUpgradeCommands: 'allowedCommands',
```

Comment:
Needs tests

Reply:
I don't think we need to test every single alias if at least one is tested

Reply:
Better to test the migrations work via tests than not.
Thw test file already has a describe block for testing migrations.

Reply:
It's wasteful and pointless to test every single pair in this object. We should focus on testing functionality, and this is not new functionality. If the lines above work, these will work too.